### ISSUE TITLE
Could not parse .eh_frame section: pointer encoding not supported

### ISSUE THREAD

**Issue by nd** [go-delve/delve#3015]
1. What version of Delve are you using (`dlv version`)?

```
Delve Debugger
Version: 1.8.2
Build: 65e03d673d31f9a6c9c2bd187a4ff2219444b16f
```

2. What version of Go are you using? (`go version`)?

```
go1.18.2 linux/amd64
```

3. What operating system and processor architecture are you using?

Ubuntu 20.04.4 LTS, Ubuntu 22.04 LTS

4. What did you do?

Debugged a program and got the error: `debugger error loading binary "/lib/x86_64-linux-gnu/libc.so.6": could not parse .eh_frame section: pointer encoding not supported 0x9b at 0x59f4`.

Is there a ticket for such a support? Is it possible to work around the error?

**Comment by aarzilli** [comment]
This shouldn't prevent the debugger from working, does it?

**Comment by nd** [comment]
Debugger works, although reportedly slower than before.

**Comment by aarzilli** [comment]
It's very strange that this would have any effect on the speed of the debugger. FWIW I'm also getting that error now but it doesn't seem slower. Are you running any specific benchmark? Describe this in greater detail.

**Comment by nd** [comment]
Not a benchmark, just a report from users. Asked for more details.

**Comment by nd** [comment]
It looks like lags were caused by something else.

**Comment by aarzilli** [comment]
Something else other than delve or something else in delve? If it's the latter I'm still interested.

**Comment by nd** [comment]
I think it was caused by IDE, not delve.

**Comment by renan-bg** [comment]
Hey, guys! I also got this error while experimenting with delve for the first time. 

1. What version of Delve are you using (`dlv version`)?
`Version: 1.8.3
Build: $Id: f92bb46b82b3b92d79ce59c4b55eeefbdd8d040c`
2. What version of Go are you using? (`go version`)?
`go version go1.18.2 linux/amd64`
3. What operating system and processor architecture are you using?
`Ubuntu 20.04.4 LTS`

I tested it on the command line, although I plan using it with Emacs.
The error: `error layer=debugger error loading binary "/lib/x86_64-linux-gnu/libc.so.6": could not parse .eh_frame section: pointer encoding not supported 0x9b at 0x5640`
